src/domain/occurrence/enrolmentDetails/EnrolmentDetails.tsx

```diff
@@ -33,12 +35,23 @@ import styles from './enrolmentDetails.module.scss';
 import EnrolmentInfoRow from './EnrolmentInfoRow';
 import { getNotificationInfoText } from './utils';
 
-const EnrolmentDetails: React.FC<{
+interface EnrolmentDetailsProps {
```

Comment:
Nice that you created interface to props types. It's more readable to me like this